server/index.js

```diff
@@ -0,0 +1,110 @@
+const express = require("express");
+const http = require("http");
+const socketIo = require("socket.io");
+
+const app = express();
+const server = http.createServer(app);
+const io = socketIo(server);
+
+let users = {};
+let files = {};
+
+io.on("connection", (socket) => {
+  users[socket.id] = {
+    cursorPosition: null,
+    name: "Anonymous",
+    filePosition: "",
+    color: random_rgba(),
+  };
+
+  socket.on("joinSession", async (sessionId) => {
+    await socket.join(sessionId);
+    console.log(`Socket ${socket.id} joined room ${sessionId}`);
```

Comment:
should we remove these before merging?

Reply:
Actually I feel like these are very useful for logs to debug on the Live Interview server. They're equivalent to `log.info` in the Python code.